main.scss

```
@@ -5,3 +5,4 @@
 @import './src/components/bottom/team-trial/main';
 @import './src/components/bottom/brexit/main';
 @import './src/components/bottom/tech-scroll-asia/main';
+@import './src/components/bottom/b2b-trial-mobile/main';
```

Did you miss this one?
```suggestion
@import './src/components/bottom/app-promo-mobile/main';
```

---

yep 😄

---

done 👍